frontend/src/components/Toast/ErrorToast.jsx

```diff
@@ -0,0 +1,18 @@
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+import React from 'react'
+
+const notifyError = (message, id, autoClose) => {
+    toast.error(message, {toastId: id || undefined, autoClose: autoClose !== undefined ? autoClose : 3000} );
```

Comment:
also, I believe `react-toastify` generates ids by itself and we do not really need custom ids in our use-case, unless I'm missing something (?)